code/modules/research/devicecrafting/obj_stuff.dm

```diff
@@ -0,0 +1,29 @@
+/obj
```

Comment:
it's so I can have a structure/machine device frame and a handheld frame without snowflaking/copypaste

what the fuck we don't absolute path vars